module/zfs/arc.c

```diff
@@ -468,6 +468,24 @@ static int zfs_arc_prune_task_threads = 1;
 /* Used by spa_export/spa_destroy to flush the arc asynchronously */
 static taskq_t *arc_flush_taskq;
 
+/*
+ * Controls the number of ARC eviction threads.
+ * Possible values:
+ * 0  (disabled) parallel eviction threads are not used.
+ * 1  (auto) compute the number of threads using a logarithmic formula.
+ * 2+ (manual) set the number manually, limited by zfs_arc_evict_threads_max.
+ */
+static uint_t zfs_arc_evict_threads = 1;
+
+/*
+ * The mumber of allocated ARC eviction threads. This limits the maximum value of
```

Comment:
The line is too long, plus "mumber" typo.

Reply:
Fixed.